modules/router-store/src/router_store_module.ts

```diff
@@ -35,6 +42,7 @@ import {
   SerializedRouterStateSnapshot,
 } from './serializers/default_serializer';
 import { MinimalRouterStateSerializer } from './serializers/minimal_serializer';
+import { ACTIVE_RUNTIME_CHECKS } from '@ngrx/store';
```

Comment:
Can this be added to the existing `store` imports please?

Reply:
😳 whoops, yes